week4/lesson-plan.md

```diff
@@ -9,6 +9,6 @@
 
 From [Niels Schmidt ](https://github.com/bytewiz)
 
-https://github.com/bytewiz/HackYourFuture/tree/master/src/ContextExample
+https://github.com/bytewiz/HackYourFuture/tree/main/src/ContextExample
```

Comment:
This external resource only works with master. But maybe @bytewiz can also change it from master to main, no?